JniBridge.bee.cs

```diff
@@ -184,24 +185,31 @@ static Jdk SetupJava()
         if (jdk != null)
             return jdk;
         var openJdk = StevedoreArtifact.UnityInternal(HostPlatform.Pick(
-            linux:   "open-jdk-linux-x64/jdk8u172-b11_4be8440cc514099cfe1b50cbc74128f6955cd90fd5afe15ea7be60f832de67b4.zip",
-            mac:     "open-jdk-mac-x64/jdk8u172-b11_01c8518f34d083b6adf57f1a3a7cff76e78dc0a2523b6332d10a0f30fcb8d308.zip",
-            windows: "open-jdk-win-x64/jdk8u172-b11_43d8964b5d97b95a87b54ed5b08b70dcb0aa4f6521a1dad4caba958e44c205b9.zip"
+            linux:   "open-jdk-linux-x64/jdk11.0.14.1-1_c17a2ae6fe1b4281fb613fed32027cf93e0251795387941bd8c1fcb6c74f9db4.zip",
+            mac:     "open-jdk-mac-x64/jdk11.0.14.1-1_236fc2e31a8b6da32fbcf8624815f509c51605580cb2c6285e55510362f272f8.zip",
+            windows: "open-jdk-win-x64/jdk11.0.14.1-1_85218201fea144521d643808d167605d6d46cd4fe44ee4001991a3a4b76fdd64.zip"
         ));
         return new Jdk(openJdk.Path.ResolveWithFileSystem());
     }
 
     static NPath SetupAndroidSdk()
     {
-        return GetEnv("ANDROID_SDK_ROOT");
+        var sdk = Environment.GetEnvironmentVariable("ANDROID_SDK_ROOT");
+        if (!string.IsNullOrEmpty(sdk))
+            return new NPath(sdk);
+        var sdkArtifact = StevedoreArtifact.UnityInternal(HostPlatform.Pick(
+            linux:   "android-sdk-linux-x86_64/32.0.0_786892eaffb9da632d76518abd381ad6e647d0c442e5df8a5ee83d780f1c9bba.7z",
+            mac:     "android-sdk-darwin-x86_64/32.0.0_01b3084deb3c473718ec212d6f34f44d64888e23f60c2b27194a84741dd128a3.7z",
+            windows: "android-sdk-windows-x86_64/32.0.0_5f6c83d41ac3d697823858e27db1774b1ecfe226b6e9a12e944caed0f76f824b.7z"
+        ));
+        return sdkArtifact.Path.ResolveWithFileSystem();
     }
 
-    static NPath GetEnv(string variable)
+    static NPath SetupGooglePlayServices()
     {
-        var path = Environment.GetEnvironmentVariable(variable);
-        if (string.IsNullOrEmpty(path))
-            throw new Exception($"Environment variable {variable} not set");
-        return new NPath(path);
+        // Downloaded from https://dl.google.com/android/repository/google_play_services_3265130_r12.zip
+        var gps = StevedoreArtifact.Testing("google-play-services/3265130_r12_3a601d0d9366fe05c7a56855f2c906a8d703fd297f1c0f6cfdaf27b181be284c.zip");
```

Comment:
I'll ask for the artifact to be coppied from the testing to the internal repository and I'll change Testing to UnityInternal before merging if everything here looks good.

Reply:
Changed in https://github.com/Unity-Technologies/android-jni-bridge/pull/41/commits/cf5e1294fa955d4487bbebb8b2e4e946be544bc0